index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+
+  <link rel="stylesheet" type="text/css" href="bower_components/bootstrap/dist/css/bootstrap.css">
+
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+
+  <title>Cars</title>
+</head>
+<body>
+<nav class="navbar navbar-default navbar-fixed-top">
+  <div class="container-fluid">
+       <form class="navbar-form navbar-left">
+        <div class="form-group">
+          <input type="text" class="form-control" placeholder="Enter Description" oninput="CarLot.activateEvenMoreEvents(event);" onchange="CarLot.activateMoreEvents(event);" id="searchBar">
```

Comment:
I must not have seen this before, but you definitely don't want to call functions from you html file. See if you can find a better way to accomplish this.